app/views/sites/emails/edit.html.erb

```diff
@@ -14,7 +14,7 @@
 
       <% @services.each do |service| %>
         <%= form.semantic_fields_for service do |sf| %>
-          <%= sf.input :support_email, :label => "Service #{service.name}", :input_html => { :value => service.support_email, :id => "service_#{service.id}_support_email", :name => "service_#{service.id}_support_email" } %>
+          <%= sf.input :support_email, :label => "Service #{ sanitize service.name }", :input_html => { :value => service.support_email, :id => "service_#{service.id}_support_email", :name => "service_#{service.id}_support_email" } %>
```

Comment:
Planning to add a test?

Reply:
To be honest, I don't know what should I test here... that it renders text?

Reply:
I think we can leave the tests for the form builder refactor @hallelujah suggested